packages/angular/projects/ui-angular/src/lib/components/authenticator/components/authenticator/authenticator.component.ts

```diff
@@ -54,13 +56,24 @@ export class AuthenticatorComponent implements OnInit, AfterContentInit {
       socialProviders,
       formFields,
     } = this;
-    this.authenticator.startMachine({
-      initialState,
-      loginMechanisms,
-      services,
-      signUpAttributes,
-      socialProviders,
-      formFields,
+
+    // send INIT event once machine is at 'setup' state
+    this.authenticator.subscribe(() => {
+      const { route } = this.authenticator;
+      if (!this.hasInitialized && route === 'setup') {
```

Comment:
Previously `authMachine` got stuck in `idle` state until an `Authenticator` in DOM sends an `INIT` event on mount. Now `authMachine` starts independently (see state machine changes below), and will explicitly ask for config from `Authenticator` once it's in `setup` state.

Once `Authenticator` sees that state is in `setup`, it'll send the usual `INIT` event.